skyrim-platform/src/tilted/ui/TextToDraw.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+struct TextToDraw
+{
+  TextToDraw(wchar_t* m_kString);
```

Comment:
Please do not use leave uninitialized variables (x, y, size).
Remove `size`. (seems hard to implement now)
wchar_t* => std::wstring

Important: **use these variables**
You currently ignore them in actual rendering code